interactive_engine/bin/giectl.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+#
+# interactive_engine command tool
+
+readonly SCRIPT_DIR=$(cd "$(dirname "$0")";pwd)
+readonly WORKSPACE=${SCRIPT_DIR}/../
+
+err() {
+  echo -e "${RED}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [ERROR]${NC} $*" >&2
+}
+
+warning() {
+  echo -e "${YELLOW}[$(date +'%Y-%m-%dT%H:%M:%S%z')]: [WARNING]${NC} $*" >&1
+}
+
+log() {
+  echo -e "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&1
+}
+
+
+##########################
+# Output useage information.
+# Globals:
+#   None
+# Arguments:
+#   None
+##########################
+usage() {
+cat <<END
+  A script to install dependencies of GraphScope or deploy GraphScope locally.
+
+  Usage: deploy_local [options] [command]
+
+  Options:
+
+    --prefix <path>      install prefix of GraphScope, default is /usr/local
+    --verbose            ouput the debug logging information
+    -h, --help           output help information
+
+  Commands:
+
+    install_deps         install dependencies of GraphScope
+    deploy               deploy GraphScope locally
+END
+}
+
+
+
+set_common_envs() {
+  object_id=$1
```

Comment:
object_id for COMMON env?

Reply:
please add a TODO: this function should be removed.
pid and log should go to standard path.

Reply:
removed